File: solutions/1603-running-sum-of-1d-array/solution.java
class Solution {
    public int[] runningSum(int[] nums) {
        int[] ans = new int[nums.length];
        for(int i=0;i<nums.length;i++){
            int x=0;
            for(int y=0;y<=i;y++){
                x+=nums[y];
            }
            ans[i]=x;
        }   
        return ans;      
}
}

File: solutions/1603-running-sum-of-1d-array/solution.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        sum_arr = []
        for i in range(len(nums)):
            if i!=0:
                sum_arr.append(nums[i]+sum_arr[i-1])
            else:
                sum_arr.append(nums[i])
        return sum_arr
